package.json

```
@@ -35,7 +35,7 @@
     "changelog": "changelog-maker"
   },
   "peerDependencies": {
-    "mobx-state-tree": "^3.2.1"
+    "mobx-state-tree": "^5.1.8"
```

So this change would be breaking, instead we can just make the existing range more permissive to allow for any version above:

```suggestion
    "mobx-state-tree": ">=3.2.1"
```

This change would be a patch bump instead